include/roboteam_ai/world/World.hpp

```diff
@@ -162,6 +162,8 @@ class World {
      */
     [[nodiscard]] ai::control::PositionControl *getRobotPositionController() noexcept;
 
+    void forwardWorldToBB();
```

Comment:
void functions that do not take any arguments are usually bad because it's unclear what is happening with the internal state of world. This function should probably be const, as well, as you don't want BB trajectories editing the world.